emily/handler/src/database/accessors.rs

```diff
@@ -117,25 +117,17 @@ pub async fn get_deposit_entries_by_reclaim_pubkeys_hash(
     .await
 }
 
-/// Hacky exhaustive list of all possible deposit statuses that we will iterate over in order to
+/// Hacky exhaustive list of all possible statuses that we will iterate over in order to
 /// get every deposit present.
-const ALL_DEPOSIT_STATUSES: &[Status] = &[
+/// TODO(https://github.com/stacks-sbtc/sbtc/issues/1693)
```

Comment:
nit: we use a different format for the todos usually
```suggestion
/// TODO(1693): Split Emily status enum for deposits and withdrawals
```